tremor-script/src/lexer.rs

```diff
@@ -137,6 +137,7 @@ pub(crate) fn ident_to_token(ident: &str) -> Token {
         "use" => Token::Use,
         "as" => Token::As,
         "recur" => Token::Recur,
+        "subquery" => Token::Subquery,
```

Comment:
`subquery` is a good description for what we implement but reading over this I want to toss in the thought that it might not be a good name for users.

From a developer's point of view, yes this is implemented as a subquery to another query, but is that relevant to a user? Would it be easier to digest to call it `query` instead (probably not?) or use `flow` as troy does?

Reply:
Yes - the keywords `pipeline` and `flow` are introduced in troy but `query` would be best here I think